server/server/src/config.rs

```diff
@@ -75,11 +76,48 @@ impl ServerConfig {
     }
 }
 
+#[derive(Clone)]
+pub struct MetricsConfig {
+    pub minutes_between_metrics_refresh: Duration,
+    pub millis_between_user_metrics: Duration,
+    pub millis_between_getting_pub_key_metrics: Duration,
+    pub millis_between_getting_pub_key_key_metrics: Duration,
```

Comment:
You don't need to specify the type here, `Duration` will do all the conversions. It's almost inaccurate to say `minutes` for a duration object.